src/js/common/components/RequiredLoginView.jsx

```diff
@@ -55,27 +53,29 @@ class RequiredLoginView extends React.Component {
       } else if (this.props.authRequired === 3) {
         if (this.props.userProfile.accountType === 3) {
           if (this.props.userProfile.status === 'SERVER_ERROR') {
-            // If va_profile is null, the system is down and we will show system down message.
+            // If va_profile is null, show a system down message.
             view = <SystemDownView messageLine1="Sorry, our system is temporarily down while we fix a few things. Please try again later."/>;
           } else if (this.props.userProfile.status === 'NOT_FOUND') {
-            // If va_profile is "not found", we cannot find you in our system and we will show a, we can't find you message.
+            // If va_profile is "not found", show message that we cannot find the user
+            // in our system.
             view = <SystemDownView messageLine1="We couldn't find your records with that information." messageLine2="Please call the Vets.gov Help Desk at 1-855-574-7286. We're open Monday‒Friday, 8:00 a.m.‒8:00 p.m. (ET)."/>;
           } else {
-            // If there is something in the va_profile attribute, continue on to check if this user can use this specific service.
+            // If va_profile has any other value, continue on to check if this user can
+            // use this specific service.
             if (this.state.isServiceAvailableForUse) {
-              // If you have the required service show the application view.
+              // If you have the required service, show the application view.
               view = this.props.children;
             } else {
-              // If you do not have the required service in your `userServices` array then we will show the component but pass a prop to let them know that you don't have any data. Only passes prop on React components (functions) and not elements like divs so that React does not throw a warning
+              // If the required service is not availble, the component will still be rendered,
+              // but we pass an `isDataAvailable` prop to child components indicating there is
+              // no data. (Only add this prop to React components (functions), and not ordinary
+              // DOM elements.)
               view = React.Children.map(this.props.children,
                 (child) => {
                   let props = null;
                   if (typeof child.type === 'function') {
-                    props = {
-                      isDataAvailable: this.state.isServiceAvailableForUse,
-                    };
+                    props = { isDataAvailable: false };
```

Comment:
Is this right? Defaulting to `false` instead of `this.state.isServiceAvailableForUse` basically tells child components that the user is LOA3 but does not have the necessary MVI attributes to use said resource.